tasks.py

```diff
@@ -3282,3 +3283,84 @@ def generate_pypi_index(context):
 """ % "\n".join(links)
     s3.Object(PYPI_S3_BUCKET, 'ai2thor/index.html').put(Body=ai2thor_index, ACL='public-read', ContentType='text/html')
 
+@task
+def ci_test_utf(context):
+    s3 = boto3.resource("s3")
+    results_path, results_logfile = test_utf(context)
+    
+    for l in [results_path, results_logfile]:
+        key = "builds/" + os.path.basename(l)
+        with open(l) as f:
+            s3.Object('ai2-prior-erick-test', key).put(
+                Body=f.read(), ContentType="text/plain"
+                #ACL=acl, 
+            )
+
+@task
+def test_utf(context):
+    project_path = os.path.join(os.getcwd(), 'unity')
+    commit_id = git_commit_id()
+    test_results_path = os.path.join(project_path, 'utf_testResults-%s.xml' % commit_id)
+    logfile_path = os.path.join(os.getcwd(), 'thor-testResults-%s.log' % commit_id)
+
+    command = (
+        "%s -runTests -testResults %s -logFile %s -testPlatform PlayMode -projectpath %s "
```

Comment:
we should probably pass width and height here, for consistent visibility tests, like we do when normally running THOR from python

Reply:
We aren't able to pass width/height on the command line or change it using `Screen.SetResolution`, but I did update the ProjectSettings to set the default width/height to 300/300.